ci/cron/src/Main.hs

```diff
@@ -183,12 +184,20 @@ fetch_gh_paginated url = do
                 (_, _, _, [url, rel]) -> (rel, url)
                 _ -> fail $ "Assumption violated: link header entry did not match regex.\nEntry: " <> l
 
-data PreVersion = PreVersion { prerelease :: Bool, tag :: Version }
-instance JSON.FromJSON PreVersion where
-    parseJSON = JSON.withObject "PreVersion" $ \v -> PreVersion
-        <$> v JSON..: "prerelease"
-        <*> let json_text = v JSON..: "tag_name"
-            in (version . Text.tail <$> json_text)
+data GitHubRelease = GitHubRelease { prerelease :: Bool, tag :: Version, assets :: [String] }
+  deriving Show
+instance JSON.FromJSON GitHubRelease where
+    parseJSON = JSON.withObject "GitHubRelease" $ \v -> GitHubRelease
+        <$> (v JSON..: "prerelease")
+        <*> (version . Text.tail <$> v JSON..: "tag_name")
+        <*> (do
+              assets <- (v JSON..: "assets")
```

Comment:
```suggestion
              assets <- v JSON..: "assets"
```
unfortunately hlint still hates parentheses :disappointed:

Reply:
I'll fix that. 🙂